CONTRIBUTING.md

```diff
@@ -238,7 +238,9 @@ Read the [Test262 Technical Rationale Report](https://github.com/tc39/test262/wi
 
 ## Test Environment
 
-Each test case is run in a fresh JavaScript environment; in a browser, this will be a new &lt;iframe&gt;; for a console runner, this will be a new process.  The test harness code is loaded before the test is run unless the test file has the `raw` flag.  The test harness defines the following helper functions:
+Each test case is run in a fresh JavaScript environment; in a browser, this will be a new &lt;iframe&gt;; for a console runner, this will be a new process.
+
+Before the test is executed (and unless the test uses the `raw` frontmatter flag), the test runner will evaluate a number of files in the `harness/` directory. At a minimum, this procedure will define the following functions:
```

Comment:
The "at a minimum" part is slightly ambiguous. It's the minimum when raw flag is not present. I tried to rephrase the same content above to resolve this, WDYT?

```suggestion
The test runner must evaluate the required files from the `harness/` directory before the test is executed. Unless the `raw` frontmatter flag is present, this procedure will define at least the following functions:
```